internal/consul/registration.go

```diff
@@ -131,6 +131,15 @@ func (s *ServiceRegistry) register(ctx context.Context) error {
 			Interval:                       serviceCheckInterval,
 			DeregisterCriticalServiceAfter: serviceDeregistrationTime,
 		}},
+		Proxy: &api.AgentServiceConnectProxyConfig{
+			Config: map[string]interface{}{
+				"envoy_gateway_bind_addresses": map[string]interface{}{
+					"__global": map[string]interface{}{
+						"address": "0.0.0.0",
+					},
```

Comment:
As far as I can tell from the Consul code this name (`"__global"`) is ignored and, as mentioned, despite the docs saying to specify a `"port"` value, it's not really used anywhere in the listener generation code 🤷 .